packages/core/scripts/setup/patches/common.ts

```diff
@@ -0,0 +1,85 @@
+import EmbraceLogger from '../../../src/logger';
+import { FileUpdatable } from '../../util/file';
+import { OBJECTIVEC_APP_DELEGATE } from './ios/ios.objectivec';
+import { SWIFT_APP_DELEGATE } from './ios/ios.swift';
+
+// TODO refactor this
```

Comment:
what change would be needed here?

Reply:
I wonder if we can use one a "static" approach. something like 
EmbraceLogger.warn|info|etc. without create an instance every file/every call